Source/Other/Extensions/DateExtension.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 
 extension Date {
 	
-	public func format(_ format: String, locale: Locale, timeZone: TimeZone) -> String {
+	public func string(_ format: String, locale: Locale = .current, timeZone: TimeZone = .current) -> String {
```

Comment:
I prefer `string(format:` because then you call it `Date().string(format: "dd.mm.yyyy")` rather than currently `Date().string("dd.mm.yyyy")` - what do you think?

Reply:
yeah sounds better